src/amenities/amenities_server/Program.cs

```diff
@@ -0,0 +1,25 @@
+var builder = WebApplication.CreateBuilder(args);
```

Comment:
Enable CORS so React and Angular can make cross-origin requests
1. Set policy
```c#
builder.Services.AddCors(options =>
{
    options.AddPolicy("AllowAll",
        builder =>
        {
            builder.AllowAnyOrigin()      // Allow requests from any origin
                   .AllowAnyMethod()      // Allow any HTTP method (GET, POST, PUT, DELETE, etc.)
                   .AllowAnyHeader();     // Allow any headers in the request
        });
});
```
2. Then enable routing & apply CORS policy
```c#
app.UseRouting(); 
app.UseCors("AllowAll");
```

Reference:
https://learn.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-8.0